src/components/map/leaflet-areaselect.js

```diff
@@ -14,13 +14,15 @@ L.AreaSelect = L.Class.extend({
     minHorizontalSpacing: 30,
     minVerticalSpacing: 30,
     keepAspectRatio: false,
+    bbox: null,
```

Comment:
What is this needed for?

Reply:
this was a stray default from work on #388 .  I've removed it